cli/src/update/unix.rs

```diff
@@ -57,7 +58,7 @@ impl Default for ApplicationUpdater {
 
 /// Generic function for fetching data via HTTP GET.
 async fn http_get(url: &str) -> anyhow::Result<reqwest::Response> {
-    let client = Client::builder().user_agent("phylum-cli").build()?;
+    let client = Client::builder().user_agent(USER_AGENT.as_str()).build()?;
```

Comment:
I discovered #665 when modifying this line. We should fix that in the future, but I didn't want to crowd this PR.